src/App.tsx

```diff
@@ -1,28 +1,53 @@
 /* eslint-disable jsx-a11y/label-has-associated-control */
 /* eslint-disable jsx-a11y/control-has-associated-label */
-import React from 'react';
-import { UserWarning } from './UserWarning';
-import { USER_ID } from './api/todos';
+import React, { useEffect, useState } from 'react';
+import { getTodos } from './api/todos';
+import { Todo } from './types/Todo';
+import cn from 'classnames';
+import { ErrorNotification } from './components/ErrorNotification';
+import { Filters } from './types/Filters';
+import { filterTodos } from './utils/todoUtils';
 
 export const App: React.FC = () => {
-  if (!USER_ID) {
-    return <UserWarning />;
-  }
+  const [todos, setTodos] = useState<Todo[]>([]);
+  const [error, setError] = useState<string>('');
+  const [filter, setFilter] = useState<Filters>(Filters.All);
+
+  const clearError = () => setError('');
+
+  useEffect(() => {
+    const loadTodos = async () => {
+      try {
+        clearError();
+        const loadedTodos = await getTodos();
+
+        setTodos(loadedTodos);
+      } catch {
+        setError('Unable to load todos');
+      }
+    };
+
+    loadTodos();
```

Comment:
```suggestion
    (async () => {
      try {
        clearError();
        const loadedTodos = await getTodos();

        setTodos(loadedTodos);
      } catch {
        setError('Unable to load todos');
      }
    })();
```

https://developer.mozilla.org/en-US/docs/Glossary/IIFE